docs/assertions-custom.md

```diff
@@ -0,0 +1,209 @@
+# User defined test function
+
+In addition to builtins assertion function, PipeRider also supports adding customized assertion function. User defined
```

Comment:
should we change to assertion class or instance?

Reply:
won't fix